InferenceSystem/src/hls_utils/gh_utils.py

```diff
@@ -0,0 +1,12 @@
+import requests
```

Comment:
I'd suggest putting this file in a different location (maybe https://github.com/orcasound/aifororcas-livesystem/tree/main/InferenceSystem/src/github_utils/github_utils.py) and renaming it to github_utils.py to be more explicit since GH is not a well-known acronym for Github (afaik?).